Jenkinsfile

```diff
@@ -479,6 +479,10 @@ pipeline {
             name: "TARGET_GFX94X",
             defaultValue: false,
             description: "")
+        booleanParam(
+            name: "TARGET_GFX94X_DBSYNC",
```

Comment:
[Recommendation] @cderb AFAICS this is workaround so I recommend:
- moving it to the end of the list of params
- renaming this to `WORKAROUND__TARGET_GFX94X__ALWAYS_ENABLE_EVEN_IF_SHORTAGE_OF_HW`, so it becomes cleaner that:
  - This is W/A for some temporary problem
  - This var enables a subset of tests on gfx94x even if all other tests are disabled for this target due to a lack of HW.